### DISCUSSION TITLE
Should pull request review comments support GHA rerun failed test?

### DISCUSSION THREAD

**Discussion by zjuwangg** [apache/gluten#8275] [General]
I noticed that now Pull Request test has the risk of failure caused by network problem, should we provided a convenient way such comments `rerun failed test` to trigger the CI again instead of pushing a empty commit?

**Comment by zhztheplayer** [comment]
> a convenient way such comments rerun failed test to trigger the CI

Some thoughts so far:

1. Do we need a bot for doing such?
2. How do we block a first-time contributor from triggering CI using comments?

**Comment by zjuwangg** [comment]
Another observation may not relate to this topic:
I also noticed that https://archive.apache.org/ has a limit on download archive
`Do note that heavy use of this service will result in immediate throttling of your download speeds to either 12 or 6 mbps for the remainder of the day, depending on severity. Continuous abuse (to the tune of more than 40 GB downloaded per week) will cause an [automatic ban](https://infra.apache.org/infra-ban.html), so please tune your services to this fact.`


I don't know whether our GHA will be affected by this restriction. If so, we may need install spark in docker image or download from other dist mirror?

**Reply by FelixYBW** [reply]
what's the limit of docker image size? can we install all packages into the docker image?

**Reply by FelixYBW** [reply]
Or save the images to github like velox's ccache
